.github/actions/setup-rust/action.yml

```diff
@@ -52,6 +52,12 @@ runs:
           echo CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse >> $GITHUB_ENV
         fi
 
+    - shell: bash
+      run: |
+        : install mold linker
+        sudo apt update
+        sudo apt install -y mold
```

Comment:
`sudo: command not found`

This fails the CI

Reply:
This also doesn't work for windows or mac